spec/Cabin/Infotainment.vspec

```diff
@@ -120,6 +120,14 @@ Navigation.GuidanceVoice:
   description: Navigation guidance state that was selected.
   comment: ETC indicates a voice alternative not covered by the explicitly listed alternatives.
 
+Navigation.MapScaleMode:
+  datatype: string
+  type: actuator
+  allowed: ['SCALE_FIXED_MODE', 'AUTO_SCALE_MODE']
```

Comment:
Do we expect more alternatives in the future? if not an alternative solution could be a boolean signal like `IsAutoScaleMode` (and if false then fixed mode is used). But this approach work for me as well.

Reply:
@erikbosch Thanks for valuable comment! I think IsAutoScaleMode is enough. I am going to revise the PR based on the consensus.

Reply:
@erikbosch Could you review updated one? Any comment on this?